Formula/gst-plugins-bad.rb

```diff
@@ -5,6 +5,7 @@ class GstPluginsBad < Formula
   stable do
     url "https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.8.3.tar.xz"
     sha256 "7899fcb18e6a1af2888b19c90213af018a57d741c6e72ec56b133bc73ec8509b"
+    revision 1
```

Comment:
Ok, I read "inserted underneath the homepage/url/sha block." in https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md